api/bucket.go

```diff
@@ -288,7 +280,7 @@ func (h *BucketHandler) delete(ctx *gin.Context, owner *model.BucketOwner) {
 	path := pathlib.Join(
 		owner.Bucket,
 		rPath)
-	err := os.RemoveAll(path)
+	err := nas.RmDir(path)
```

Comment:
Using nas.RmDir() which uses `rm -rf` because it's 12x faster.  RemoveAll() is super slow.